Framework/Core/include/Framework/Dispatcher.h

```diff
@@ -16,6 +16,7 @@
 #ifndef ALICEO2_DISPATCHER_H
 #define ALICEO2_DISPATCHER_H
 
+#include <Monitoring/Monitoring.h>
```

Comment:
Actually it would be a good idea to forward declare this rather than doing a full include.

Reply:
Indeed, I will have a look at all the includes in Data Sampling, maybe i will find something else to fix as well.